x-pack/plugins/maps/public/classes/sources/es_search_source/es_search_source.tsx

```diff
@@ -599,7 +602,10 @@ export class ESSearchSource extends AbstractESSource implements IMvtVectorSource
     searchSource.setField('query', query);
     searchSource.setField('fieldsFromSource', this._getTooltipPropertyNames());
 
-    const resp = await searchSource.fetch({ legacyHitsTotal: false });
+    const resp = await searchSource.fetch({
+      legacyHitsTotal: false,
+      executionContext: makePublicExecutionContext('es_search_source:all_doc_counts'),
```

Comment:
This request is not for all_doc_counts, but instead, requests properties for tooltip. How about changing to `es_search_source:load_tooltip_properties`